### DISCUSSION TITLE
Deduplication of "black" videos

### DISCUSSION THREAD

**Discussion by DeltaTango69** [immich-app/immich#10859] [Q&A] [answered]
### The bug

All videos with a black frame are recognized as duplicate and will be trashed. I think videos in general should not be deduplicated!

### The OS that Immich Server is running on

Windows 10

### Version of Immich Server

V1.107.2

### Version of Immich Mobile App

v1.08.1

### Platform with the issue

- [ ] Server
- [X] Web
- [ ] Mobile

### Your docker-compose.yml content

```YAML
irrelevant
```

### Your .env content

```Shell
irrelevant
```

### Reproduction steps

```bash
see above
```

### Additional information

![Screenshot 2024-07-04 180204](https://github.com/immich-app/immich/assets/82575543/4dce886b-330f-4bd6-a4fd-579204d4ace5)

**Comment by mertalev** [comment, marked as answer]
We improved video thumbnails recently to choose a non-black thumbnail when possible. You can regenerate thumbnails on all assets to update them, run smart search on all assets and finally run duplicate detection again.

**Comment by DeltaTango69** [comment]
The problem is not only by black videos. Also the videos are taken at totally different dates and has totally different names - this could never be duplicates?

**Reply by mertalev** [reply]
Different metadata or names doesn't mean it isn't a visual duplicate. The same image uploaded to different places can end up with different data, etc.
